CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.30.0] - 2020-03-26
```

Comment:
So because this changes will go into the next release (but are unreleased as now), we just use `[Unreleased]` placeholder here. At the time of the release, `[Unreleased]` gets replaced with the release version and date.
```suggestion
## [unreleased]
```